src/spotbugs/spotbugs-excludes.xml

```diff
@@ -0,0 +1,14 @@
+<FindBugsFilter>
+  <!-- disable all findsecbugs findings -->
+<!--    <Match>
+        <Bug category="SECURITY"/>
+    </Match>  -->
+  <Match>
```

Comment:
This and the next one are reasonable in Jenkins. They're not needed specifically for this plugin except for the fake issue added here. They're needed in other modules such as jenkinsci/jenkins.